Sources/StreamChatUI/CommonViews/Suggestions/ChatSuggestionsViewController.swift

```diff
@@ -84,47 +91,26 @@ open class _ChatSuggestionsViewController<ExtraData: ExtraDataTypes>: _ViewContr
             options: [.new],
             changeHandler: { [weak self] collectionView, change in
                 DispatchQueue.main.async {
-                    guard let self = self else { return }
+                    guard let self = self, let newSize = change.newValue else { return }
 
                     // NOTE: The defaultRowHeight height value will be used only once to set visibleCells
                     // once again, not looping it to 0 value so this controller can resize again.
                     let cellHeight = collectionView.visibleCells.first?.bounds.height ?? self.defaultRowHeight
 
-                    guard let newSize = change.newValue,
-                          newSize.height < cellHeight * self.numberOfVisibleRows
-                    else {
-                        self.view.frame.size.height = cellHeight * self.numberOfVisibleRows
-                        self.updateViewFrame()
-                        return
-                    }
-                    self.view.frame.size.height = newSize.height
-                    self.updateViewFrame()
+                    let newHeight = min(newSize.height, cellHeight * self.numberOfVisibleRows)
+                    self.heightConstraints.constant = newHeight
                 }
             }
         )
-        updateContent()
```

Comment:
I probably do not understand deletion of this as well, I knew this had some meaning, however I should call `updateContentIfNeeded`

Reply:
updateContent is called automatically